lib/js/components/Icon/Icon.stories.mdx

```diff
@@ -1,81 +0,0 @@
-import { Meta, Story, Props, IconGallery, IconItem } from '@storybook/addon-docs';
-import WnlIcon, { ICON_SIZES, ICONS } from 'design-system/lib/js/components/Icon';
-
-<Meta title="components/Icon" component={WnlIcon} />
-
-# Icon
-
-<Story
-	name="interactive"
-	parameters={{
-		design: {
-			type: 'figma',
-			url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=1552%3A34963',
-		},
-	}}
-	args={{
-		size: ICON_SIZES.MEDIUM,
-		icon: 'HEAD_WITH_QUESTION_MARK',
-		spinning: false,
-		touchable: false,
-		rotation: null,
-		flippedVertical: false,
-		flippedHorizontal: false,
-	}}
-	argTypes={{
-		size: {
-			control: { type: 'select', options: Object.values(ICON_SIZES) },
-			defaultValue: ICON_SIZES.MEDIUM,
-		},
-		icon: {
-			control: { type: 'select', options: Object.keys(ICONS) },
-			defaultValue: 'HEAD_WITH_QUESTION_MARK',
-		},
-		rotation: {
-			control: { type: 'select', options: [null, 90, 180, 270] },
-			defaultValue: null,
-		},
-	}}
->
-	{(args) => {
-		return {
-			components: { WnlIcon },
-			props: Object.keys(args),
-			created() {
-				this.ICONS = ICONS;
-			},
-			template:
-				'<div class="iconList__singleIcon"><wnl-icon :icon="ICONS[icon]" :size="size" :touchable="touchable" :spinning="spinning" :rotation="rotation" :flipped-vertical="flippedVertical" :flipped-horizontal="flippedHorizontal" /></div>',
-		};
-	}}
-</Story>
-
-<Story
-	name="all icons"
```

Comment:
was this story removed on purpose?

Reply:
There was a problem with messy order of components, UX pointed out again that this bothers them. It was also annoying for me, so I decided to migrate by the way rest of components to `ts`. I have 4 components left, which I will add to this PR

Reply:
I misunderstood Your comment. Right, I added missing story.